README.md

```diff
@@ -46,6 +33,18 @@ This deployment is available under the _kafka-statefulsets_ folder and provides
 
 To conveniently deploy a StatefulSet to OpenShift a [Template is provided](kafka-statefulsets/resources/openshift-template.yaml), this could be used via `oc create -f kafka-statefulsets/resources/openshift-template.yaml` so it is present within your project. To create a whole Kafka StatefulSet use `oc new-app barnabas`.
 
+## Kafka in-memory
+
+Kafka in-memory deployment is just for development and testing purpose and not for production. It is designed the same way as the Kafka Stateful Sets deployment. The only difference is that for storing broker information (Zookeeper side) and topics/partitions (Kafka side), an _emptyDir_ is used instead of Persistent Volume Claims. This means that its content is strictly related to the pod life cycle (deleted when the pod goes down). However for development and testing it might be easier to use the in-memory deployment because it doesn't need to provision the persistent volumes needed for the stateful set deployment.
+
+This deployment is available under the _kafka-inmemory_ folder and provides following artifacts :
+
+* resources : provides all YAML configuration files for setting up services and deployments
+
+### Deploying to OpenShift
+
+To conveniently deploy a StatefulSet to OpenShift a [Template is provided](kafka-inmemory/resources/openshift-template.yaml), this could be used via `oc create -f kafka-inmemory/resources/openshift-template.yaml` so it is present within your project. To create a whole Kafka in-memory cluster use `oc new-app barnabas`.
```

Comment:
Would be better as a numbered list of step-by-step instructions?

Reply:
Agreed - changed both for in-memory and stateful set.